ppr-ui/src/components/search/SearchBarList.vue

```diff
@@ -130,9 +133,10 @@ export default defineComponent({
         }
         return allSearchTypes
       }),
+      isPPROnly: computed((): boolean => hasPprRole.value && !(hasMhrRole.value && getFeatureFlag('mhr-ui-enabled'))),
       displayItems: [],
       displayGroup: {
-        1: false,
+        1: !(hasPprRole.value && (hasMhrRole.value && getFeatureFlag('mhr-ui-enabled'))),
```

Comment:
How does this resolve the timing issue?

Also long term, keep in mind this flag will eventually be `true` always and we will remove this flag. Would this code remain true then? Shouldn't we be able to dictate these default values without a flag?

You seem opposed to my previous ideas of implementation, which is totally fine but i can't agree with use a 'temporary' feature flag to determine multiple values and conditions in this file. We should use the flags sparingly.

Reply:
I'm not opposed, just trying to find a solution that works for all scenarios/roles/flags.

Reply:
We need to use `mhr-ui-enabled` otherwise the dropdown behaviour won't be consistent.

Reply:
Please elaborate.

Reply:
Messaged you offline.